### DISCUSSION TITLE
auth-helper for SolidJS

### DISCUSSION THREAD

**Discussion by MariuzM** [supabase/supabase#16391] [Feature Requests]
Please please, i find SolidStart much faster than Next.js and network responses are also faster not only UI. So would like to continue with SolidStart but really need auth-helper, mostly to fix the 
```
No storage option exists to persist the session, which may result in unexpected behavior when using auth.
        If you want to set persistSession to true, please provide a storage option or you may set persistSession to false to disable this warning.
```

**Comment by silentworks** [comment]
There is some code for one [here](https://github.com/caseybaggz/auth-helpers/tree/feat/make-solid-start/packages/solidstart). I asked the developer to publish it as it's own package but not sure if they got around doing this in the end.
